lib/diagnostic.rb

```diff
@@ -108,10 +108,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'steve = Hash.new; steve[:favorite_number] = 12; steve[:first_name] = 'Steve';'
```

Comment:
We wanted the shorthand syntax:

`'person = {first_name: "Steve", favorite_number: 12}'`